scene/3d/xr_hand_modifier_3d.cpp

```diff
@@ -253,8 +255,11 @@ void XRHandModifier3D::_update_skeleton() {
 				skeleton->set_bone_pose_rotation(joints[joint].bone, Quaternion(relative_transform.basis));
 			}
 
-			// Transform to the skeleton pose.
-			set_transform(transforms[XRHandTracker::HAND_JOINT_PALM]);
+			// Transform to the skeleton pose. This uses the HAND_JOINT_PALM position without skeleton-scaling, as it
+			// must be positioned to match the physical hand location. It is scaled with the world space to match
+			// the scaling done to the camera and eyes.
+			set_transform(
+					tracker->get_hand_joint_transform(XRHandTracker::HAND_JOINT_PALM).scaled(Vector3(ws, ws, ws)));
```

Comment:
```suggestion
					tracker->get_hand_joint_transform(XRHandTracker::HAND_JOINT_PALM) * ws);
```
Equivalent I believe

Reply:
Good catch - I hadn't checked the operators.